internal/client/api_account.go

```diff
@@ -167,15 +176,7 @@ func (c *Client) ApiReadAccounts(ctx context.Context, accFilter AccountFilter) (
 			accFilter.APIFormOfFilter(), err)
 	}
 
-	accs := struct {
-		UsersCount    int    `json:"usersCount"`
-		OrgsCount     int    `json:"orgsCount"`
-		ResourceCount int    `json:"resourceCount"`
-		NextPageStart string `json:"nextPageStart"`
-
-		Accounts []ResponseAccount `json:"accounts"`
-	}{}
```

Comment:
I made this an explicit public struct, so that we don't have to maintain two duplicate structs